src/pages/lib-v1/auth/manageusers/q/platform/[platform].mdx

```diff
@@ -1,4 +1,109 @@
 export const meta = {
   title: `Password & user management`,
-  description: `Learn more about how to handle common password and user management scenarios. E.g. password reset, account recovery etc.`,
-};
\ No newline at end of file
+  description: `Learn more about how to handle common password and user management scenarios. E.g. password reset, account recovery etc.`
+};
+
```

Comment:
Should this content be available to all platforms? If not, all content can be wrapped in an `<InlineFilter>` tag specifying which platforms it is applicable to. 

<img width="1089" alt="Screenshot 2023-09-22 at 3 18 14 PM" src="https://github.com/aws-amplify/docs/assets/30757403/86970a44-91ec-43b3-9667-e36fef396d2a">